R/ops-division.r

```diff
@@ -89,7 +89,10 @@ divisible_period <- function(per, anchor) {
 
 
 divide_period_by_period <- function(per1, per2) {
-  message("estimate only: convert to intervals for accuracy")
+  verbose <- getOption("lubridate.verbose")
+  if( isTRUE(verbose) )
+    message("estimate only: convert to intervals for accuracy")
```

Comment:
I think this message could be removed altogether. All similar messages have been removed in 1.7.0. Plus I think that "convert to intervals" just doesn't make sense in that context.

Reply:
Even better: `if   (isTRUE(getOption("lubridate.verbose", TRUE))`

Reply:
I don't think it's better; that option is logical. Also the default should be FALSE to comply with the usage in other places.